index.js

```diff
@@ -130,7 +154,7 @@ exports.flattenThread = function (thread) {
       if (addedIds.has(msg.key))
         return // skip duplicates
       // insert if a mention to its parent
-      if (msg.value.content.type == 'post' && isaMentionTo(msg, parent))
+      if (thingsThatArePosts.indexOf(msg.value.content.type !== -1) && isaMentionTo(msg, parent))
```

Comment:
wait, how is this one a bug? if a `post` is 'replaced' with a `post-edit`, don't we want it to be weave-able the same way?